packages/schema/src/schema.graphql

```diff
@@ -16,6 +16,11 @@ implementation(drupal): custom.webform::url
 """
 directive @webformIdToUrl(id: String!) on FIELD_DEFINITION
 
+"""
+implementation(drupal): custom.contact::create
+"""
+directive @createContact(contact: ContactInput) on FIELD_DEFINITION
```

Comment:
To generalize this, we could just remove the argument, and make the directive simply grab the first field argument.

Reply:
this should be fixed. There is a directive now called createSubmission() which can create submissions to any webforms.